### ISSUE TITLE
BlockBundle dependency

### ISSUE THREAD

**Issue by winzou** [sonata-project/SonataMediaBundle#795]
Hi,

I have an issue installing this MediaBundle v2.3.1.
I followed instructions given here: https://sonata-project.org/bundles/media/master/doc/reference/installation.html

I have this error:

```
ClassNotFoundException in MediaBlockService.php line 34:
Attempted to load class "BaseBlockService" from namespace "Sonata\BlockBundle\Block".
Did you forget a "use" statement for another namespace?
```

Indeed this class is depending on BlockBundle that I don't have (not listed as a dependency of MediaBundle): https://github.com/sonata-project/SonataMediaBundle/blob/master/Block/MediaBlockService.php#L15

I have tried to install BlockBundle but it leads to another error. I would prefer not to install it if possible. Did I miss something in the configuration?

My config.yml:

```
sonata_media:
db_driver: doctrine_orm
class:
    media:             MediaBundle\Entity\Media
    gallery:           MediaBundle\Entity\Gallery
    gallery_has_media: MediaBundle\Entity\GalleryHasMedia
default_context: default
contexts:
    default:
        providers:
            - sonata.media.provider.dailymotion
            - sonata.media.provider.youtube
            - sonata.media.provider.image
            - sonata.media.provider.file
        formats:
            small: { width: 100 , quality: 70}
            big:   { width: 500 , quality: 70}
cdn:
    server:
        path: /uploads/media
filesystem:
    local:
        directory:  %kernel.root_dir%/../web/uploads/media
        create:     false
```

Thanks a lot

**Comment by NeuralClone** [comment]
Your MediaBundle config looks ok as far as I can tell. What's the error you ended up with after you installed the BlockBundle?

**Comment by smuriu** [comment]
Hello,

Got a similar error. Resolved by installing AdminBundle. Would be nice if AdminBundle (and BlockBundle) were not hard dependencies or provided as part of a separate bundle.

**Comment by jordisala1991** [comment]
This was fixed in: #1076
